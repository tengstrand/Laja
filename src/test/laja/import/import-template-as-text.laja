#write "target/test-output/import-template-as-text.out"
My message to you is:
  #import "src/test/laja/import/file1.laja" as text
 {name}
#end