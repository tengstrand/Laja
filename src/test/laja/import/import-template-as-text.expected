My message to you is:
#set (hello = "Hello")
#set (name = "Bill")
{hello}
 NULL
